Packs/MicrosoftGraphSecurity/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Microsoft Graph Security",
     "description": "Unified gateway to security insights - all from a unified Microsoft Graph\n  Security API.",
     "support": "xsoar",
-    "currentVersion": "2.1.12",
+    "currentVersion": "2.2.0",
```

please bump the version to 2.1.13
and change the `Packs/MicrosoftGraphSecurity/ReleaseNotes/2_2_0.md` file name to `Packs/MicrosoftGraphSecurity/ReleaseNotes/2_1_13.md`
```suggestion
    "currentVersion": "2.1.13",
```